.eslintrc

```diff
@@ -13,6 +13,7 @@
   "parserOptions": {
     "ecmaVersion": 2018
   },
+  "ignorePatterns": ["examples/**/*"],
```

Comment:
currently, if the file is not ignored, this error happens when trying to make a commit (not just for the `sgidClient` file but for the entire directory)
<img width="1171" alt="image" src="https://github.com/opengovsg/sgid-client/assets/63008910/6b829cd8-eac6-445e-9ca4-533fc62701dd">

as each example app has its own ESLint file as well (generated by Next.js), I was thinking it would be easier to just omit the ESLint checking for the examples - otherwise we would have them into the root `tsconfig.json`

I also do see the merit of standardizing linting and formatting across all examples though - what do you think?

Reply:
Ah ok my bad, I didn't see any ESLint files being committed and I'm not super familiar with Next.js. I think we can leave it as is, I don't think this is super important, I was just curious haha